projects/union_path_integration/convergence_simulation.py

```diff
@@ -117,6 +116,14 @@ def doExperiment(cellDimensions,
 
   thresholds = numModules
   perModRange = float(90.0 / float(numModules))
+
+  # No need to keep track of extra points for corners and discrete methods.
+  if anchoringMethod == "corners":
+    cellCoordinateOffsets = (.0001, .5, .9999)
```

Comment:
Don't need `.5` do you?

Reply:
Also might want to add a note somewhere to indicate that the cellCoordinateOffsets don't get used if using corners or discrete.